src/features/posts/components/grids/my-posts-feed.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import { MyPostsParamsVerifier } from "@/components/my-posts-params-verfier";
+import { useGetMyPosts } from "@/features/posts/api/use-get-my-posts";
+import { Suspense } from "react";
+import { Skeleton } from "@/components/ui/skeleton";
+import { Card, CardDescription, CardHeader } from "@/components/ui/card";
+import { MyPostCard } from "@/features/posts/components/cards/my-post-card";
+import UserCard, {
+  UserCardSkeleton,
+} from "@/features/posts/components/cards/user-card";
+import { redirect } from "next/navigation";
+
+export default function MyPostsFeed() {
+  const postsQuery = useGetMyPosts();
+  const posts = postsQuery?.data?.data || [];
```

Comment:
I couldn't think of a more elegant way to access the list of posts.

Reply:
To get a return type of query, you need to extract the data from the response and return it